### PR TITLE
[STRATCONN] 3182 added dropdown for facebook capi

### PR DESCRIPTION
**Pull request by AnkitSegment**
_A summary of your pull request, including the what change you're making and why._

Added dropdown for Facebook CAPI
JIRA TICKET: https://segment.atlassian.net/browse/STRATCONN-3183?atlOrigin=eyJpIjoiOTMzMzllNWJhMTJlNGU2NDk5MDkzNmQ5OGUwMDM0ZWIiLCJwIjoiaiJ9

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

Screenshot 2023-09-25 at 4 52 12 AM

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/browser-destinations/destinations/cdpresolution/src/generated-types.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
+   * Client identifier provided by CDP Resolution [Hashed Account ID]
    */
-  endpoint: string
+  clientIdentifier: string
   /**
-   * Client identifier provided by CDP Resolution [CDP Resultion Account](https://www.cdpresolution.com/accountsettings)
+   * Identity resolution endpoint. [CDP Resolution documentation](https://www.cdpresolution.com/docs/)
    */
-  clientIdentifier: string
+  endpoint: string
```
Comment (varadarajan-tw) [inline packages/browser-destinations/destinations/cdpresolution/src/generated-types.ts:11]: Could you remove these changes as they seem unrelated to your PR?

File: packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-properties.ts [hunk 2]
```diff
@@ -178,7 +178,17 @@ export const action_source: InputField = {
   label: 'Action Source',
   description:
     'This field allows you to specify where your conversions occurred. See [Facebook documentation](https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/server-event) for supported values.',
-  type: 'string'
+  type: 'string',
+  choices: [
```
Comment (varadarajan-tw) [inline packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-properties.ts:182]: Just double checking here - Do you think this change is backward compatible?

Comment (AnkitSegment) [reply packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-properties.ts:182]: It will not effect the existing destinations as added only dropdown it shows only when we click on it.

**Review by varadarajan-tw — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
note: this was deployed 26-Sep-23
